Create lwm2m-agent-1021-9-0-simplification-of-logging-event-levels-for-lwm-2-m-device-configuration.md (#3188)
SoftwareAG/c8y-docs: content/change-logs/device-management/lwm2m-agent-1021-9-0-simplification-of-logging-event-levels-for-lwm-2-m-device-configuration.md

---
date: ""

> title: Simplification of logging event levels for LWM2M device configuration

Reviewer: ```suggestion
title: Simplified logging event levels in LWM2M device configuration
```

product_area: Device management & connectivity
change_type:

>   - value: change-QHu1GdukP
>     label: Feature

Reviewer 2: ```suggestion
  - value: change-2c7RdTdXo4
    label: Improvement
```

component:
  - value: component-1KLUzmqfe
    label: LWM2M
build_artifact:
  - value: tc-ggH2M4hf3
    label: lwm2m-agent
ticket: DM-4555
version: 1021.9.0
---

> The logging event levels are updated to simplify the device configuration. Previously available logging levels were: NONE, LIFECYCLE, TRAFFIC, FIRMWARE, and VERBOSE. It is now simplified to NONE, LIFECYCLE, and VERBOSE. TRAFFIC and FIRMWARE log events are now merged with VERBOSE and this configuration will be automatically updated in the device managed object.

Reviewer: ```suggestion
The logging event levels have been updated to simplify the LWM2M device configuration. Previously available logging levels were: NONE, LIFECYCLE, TRAFFIC, FIRMWARE, and VERBOSE. This has now been simplified to NONE, LIFECYCLE, and VERBOSE. TRAFFIC and FIRMWARE log events have been merged with VERBOSE. This configuration will be automatically updated in the device managed object.
```
